config/kcp/kustomization.yaml

```diff
@@ -1,32 +1,40 @@
 # Adds namespace to all resources.
-namespace: integration-service-system
+# namespace: integration-service-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: integration-service-
+# namePrefix: integration-service-
```

Comment:
could you rename deployment `controller-manager` to `integration-controller-manager`? That would be pretty useful for searching for deployment on physical cluster.